static/js/popover_menus.js

```diff
@@ -85,4 +150,80 @@ export function initialize() {
             compose_mobile_button_popover_displayed = false;
         },
     });
+
+    function schedule_message_to_custom_date(send_at_time) {
+        if (!send_at_time) {
+            return;
+        }
```

Comment:
Why is this conditional necessary?

Reply:
because `set_compose_box_schedule` can also return `false`.